### PR TITLE
costas gwi cats application

### PR DESCRIPTION
**Pull request by consmavr**

### CODE REVIEW

File: src/common/custom-card.component.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+import React from 'react';
+import Card from '@mui/material/Card';
+import { CustonBackdropComponent } from './custom-backdrop.component';
+
+const style = {
+    position: 'absolute' as 'absolute',
```
Comment (padelis) [inline src/common/custom-card.component.tsx:7]: Is `as` needed here?

**Review by padelis — COMMENTED** [review]
Hello consmavr! 👋  
Thanks for taking the time and effort to submit this assignment.
Could you take a look at my comments below?

File: src/services/image.storage.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+export interface IPhoto {
+    id: string;
+    url?: string;
+    width?: number;
+    height?: number;
+}
+
+export interface ICatPhoto extends IPhoto {
+    breeds: any[];
```
[the remark this answers is not in the document]
Comment (consmavr) [reply src/services/image.storage.ts:11]: Yes you are right, but first i would have to add some extra fields  in the interface  (country_code, life_span etc.) which are used in `cat-details.component.tsx`

File: src/common/custom-card.component.tsx (hunk 1 above)
Comment (consmavr) [reply src/common/custom-card.component.tsx:7]: not really just `position: 'absolute',` should be ok, just copy pasted it probably from material UI and didn't think more of it

**Pull request closed without merging**